### PR TITLE
Update readme for AWS Container Insights Receiver

### PR DESCRIPTION
**Pull request by pxaws**
**Description:** 
* update the architecture diagram
* update the metrics and attributes
* add a sample configuration

**Link to tracking Issue:** 
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2307

### CODE REVIEW

**Comment by pxaws** [comment]
@Aneurysm9 @anuraaga Could you take a look? Thanks!

File: receiver/awscontainerinsightreceiver/README.md [hunk 1]
```diff
@@ -28,200 +27,646 @@ Note the current implementation is temporarily hosted in AWS OpenTelemetry Colle
...
+| kubernetes                        |
+| container_status                  |
+| container_status_reason           |
+| container_last_termination_reason | 
+
+The attribute `container_status_reason` is present only when `container_status` is in "Waiting" or "Terminated" State. The attribute `container_last_termination_reason` is present only when `container_status` is in "Terminated" State.
 
 
+<br/><br/> 
+<br/><br/>
```
Comment (Aneurysm9) [inline receiver/awscontainerinsightreceiver/README.md:672]:
```suggestion
```

Not sure what purpose these serve.

**Review by Aneurysm9 — APPROVED** [review]

File: receiver/awscontainerinsightreceiver/README.md (hunk 1 above)
Comment (pxaws) [reply receiver/awscontainerinsightreceiver/README.md]: Removed

**Review by anuraaga — COMMENTED** [review]
This seems like more a design doc than a readme. Can we split it out to a separate doc and make this focused on how to configure the component?

**Comment by pxaws** [comment]
> This seems like more a design doc than a readme. Can we split it out to a separate doc and make this focused on how to configure the component?

It makes sense. I split the doc into readme and design doc.

File: receiver/awscontainerinsightreceiver/README.md [hunk 2]
```diff
@@ -5,223 +5,656 @@ AWS Container Insights Receiver (`awscontainerinsightreceiver`) is an AWS specif
...
+## Design of AWS Container Insights Receiver
+
+See the [design doc](./design.md)
+
+## Configuration
+Example configuration:
+```
+receivers:
+  awscontainerinsightreceiver:
+    #collection_interval: 60s
```
Comment (anuraaga) [inline receiver/awscontainerinsightreceiver/README.md:22]: I don't think we need `#` (if the intent is to indicate they're optional maybe just say in text All parameters are optional)

**Review by anuraaga — APPROVED** [review]
Thanks, the README is much cleaner. Just one nit

File: receiver/awscontainerinsightreceiver/README.md (hunk 2 above)
Comment (pxaws) [reply receiver/awscontainerinsightreceiver/README.md]: Sure. I will make the change.

**Review by mxiamxia — APPROVED** [review]

**Review by alolita — APPROVED** [review]
+1

**Pull request merged**